ext/storage_sources/s3_store/test/test_s3_connection.cpp

```diff
@@ -13,7 +13,7 @@ int TestListBuckets(const Aws::S3Crt::ClientConfiguration &config);
 int TestListObjects(const Aws::S3Crt::ClientConfiguration &config);
 
 int CleanupTestListObjects(
-  const Aws::S3Crt::ClientConfiguration &config, std::string bucketName, int totalObjects);
+  const Aws::S3Crt::ClientConfiguration &config, const std::string bucketName, const int totalObjects, const std::string prefix);
```

Comment:
We can avoid passing a copy of the string input parameters. Use a reference instead: `&`.